README.md

```diff
@@ -5,6 +5,118 @@
 This project allows you to create intentionally-vulnerable virtual
 machines for testing purposes.
 
+## Requirements ##
+
+* [AWS CLI access
+  configured](
+  https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+  on your system
+
```

Comment:
Why the blank lines between bullets?

Reply:
Thanks- I was fighting with a linter message about blank lines between lists, but it turned out to be an issue with not indenting the subsequent lines of the list items.  Anyway, I got rid of those blank lines in 8673edc.